src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -23,6 +23,8 @@
 
 using Task = System.Threading.Tasks.Task;
 
+#nullable enable
```

Comment:
I will start introducing nullability annotations throughout the dependency node.